app/client/ui/ExampleInfo.ts

```diff
@@ -7,6 +10,8 @@ export interface IExampleInfo {
   welcomeCard: WelcomeCard;
 }
 
+const t = makeT('ExampleInfo');
```

Comment:
We also need to defer translation in this file. Here's one approach that works:

Remove `makeT`, and update the interfaces to the following:
```
export interface IExampleInfo {
  id: number;
  urlId: string;
  title: KeyAndContext;
  imgUrl: string;
  tutorialUrl: string;
  welcomeCard: WelcomeCard;
}

interface WelcomeCard {
  title: KeyAndContext;
  text: KeyAndContext;
  tutorialName: KeyAndContext;
}

type KeyAndContext = [string, Record<string, string>];
```

Then, update the examples to look like this:
```
export const examples: IExampleInfo[] = [{
  id: 1,    // Identifies the example in UserPrefs.seenExamples
  urlId: 'lightweight-crm',
  title: ['Title', {context: "CRM"}],
  imgUrl: 'https://www.getgrist.com/themes/grist/assets/images/use-cases/lightweight-crm.png',
  tutorialUrl: 'https://support.getgrist.com/lightweight-crm/',
  welcomeCard: {
    title: ['WelcomeTitle', {context: "CRM"}],
    text: ['WelcomeText', {context: "CRM"}],
    tutorialName: ['WelcomeTutorialName', {context: "CRM"}],
  },
}, {
  id: 2,    // Identifies the example in UserPrefs.seenExamples
  urlId: 'investment-research',
  title: ['Title', {context: "investmentResearch"}],
  imgUrl: 'https://www.getgrist.com/themes/grist/assets/images/use-cases/data-visualization.png',
  tutorialUrl: 'https://support.getgrist.com/investment-research/',
  welcomeCard: {
    title: ['WelcomeTitle', {context: "investmentResearch"}],
    text: ['WelcomeText', {context: "investmentResearch"}],
    tutorialName: ['WelcomeTutorialName', {context: "investmentResearch"}],
  },
}, {
  id: 3,    // Identifies the example in UserPrefs.seenExamples
  urlId: 'afterschool-program',
  title: ['Title', {context: "afterschool"}],
  imgUrl: 'https://www.getgrist.com/themes/grist/assets/images/use-cases/business-management.png',
  tutorialUrl: 'https://support.getgrist.com/afterschool-program/',
  welcomeCard: {
    title: ['WelcomeTitle', {context: "afterschool"}],
    text: ['WelcomeText', {context: "afterschool"}],
    tutorialName: ['WelcomeTutorialName', {context: "afterschool"}],
  },
}];
```

Finally, in `ExampleCard.ts`, add the translate function and use it like so:
```
const t = makeT('ExampleCard');
...
const cardElem = cssCard(
  cssImage({src: example.imgUrl}),
  cssBody(
    cssTitle(t(...card.title)),
    cssInfo(t(...card.text)),
    cssButtons(
      cssLinkBtn(cssLinkIcon('Page'), t(...card.tutorialName),
        {href: example.tutorialUrl, target: '_blank'},
      ),
      // TODO: Add a link to the overview video (as popup or to a support page that shows the
      // video). Also include a 'Video' icon.
      // cssLinkBtn(cssLinkIcon('Video'), 'Grist Video Tour'),
    )
  ),
  cssCloseButton(cssBigIcon('CrossBig'),
    dom.on('click', close),
    testId('example-card-close'),
  ),
  testId('example-card'),
);
```

Reply:
Thanks for the detailed explanation!

I did as suggested in my last commit, but I have an alternative proposal which is to wrap the examples in a function, like `buildExamples`, and call `buildExamples()` where needed instead of `examples`. We wouldn't need to defer the call to `t` to a different file, which I think makes it a little harder to follow. Happy to do it if you think it's a good idea.

Reply:
> Thanks for the detailed explanation!
> 
> I did as suggested in my last commit, but I have an alternative proposal which is to wrap the examples in a function, like `buildExamples`, and call `buildExamples()` where needed instead of `examples`. We wouldn't need to defer the call to `t` to a different file, which I think makes it a little harder to follow. Happy to do it if you think it's a good idea.

Sure, that sounds good to me. Thanks.